app/decorators/service_provider_session_decorator.rb

```diff
@@ -33,6 +33,11 @@ def nav_partial
     'shared/nav_branded'
   end
 
+  def forgot_password_partial
+    'reset_password/link' if loa3?
+    'forgot_password/link'
```

Comment:
yeah its correct in my local code, but I can switch to a block if it seems clearer